docs/sources/developers/plugins/metadata.md

```diff
@@ -167,10 +167,11 @@ For data source plugins. Token authentication section used with an OAuth API.
 
 #### Properties
 
-| Property | Type              | Required | Description                                                               |
-|----------|-------------------|----------|---------------------------------------------------------------------------|
-| `params` | [object](#params) | No       | For data source plugins. Parameters for the token authentication request. |
-| `url`    | string            | No       | For data source plugins. URL to fetch the authentication token.           |
+| Property  | Type              | Required | Description                                                                                     |
+|-----------|-------------------|----------|-------------------------------------------------------------------------------------------------|
+| `params`  | [object](#params) | No       | For data source plugins. Parameters for the token authentication request.                       |
+| `url`     | string            | No       | For data source plugins. URL to fetch the authentication token.                                 |
+| `headers` | array             | No       | For data source plugins. Route headers adds HTTP headers to the authentication request. |
```

Comment:
```suggestion
| `headers` | array             | No       | For data source plugins. Route headers add HTTP headers to the proxied request.         |
```

This doesn't correspond to the description in the plugin.schema.json. 

Tip: Use the [json-schema-docs](https://github.com/grafana/json-schema-docs) tool to generate the Markdown from the schema:

```
json-schema-docs -schema ./plugin.schema.json -template metadata.md.tpl > metadata.md
```

Reply:
I ran the json-schema-docs. A couple of things to note - it updated several other things that were unrelated to my changes, but I'm going to leave them since it now matches the schema file. Also, I updated the schema to read "For data source plugins. Route headers adds HTTP headers to the token authentication request." That doesn't match what was originally recommended by @aknuds1 , but I think it more correctly matches the wording for the other token auth specific properties instead of copying the description from the route headers.